x/evm/ante/sig_test.go

```diff
@@ -0,0 +1,79 @@
+package ante
+
+import (
+	"encoding/hex"
+	"math/big"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/sei-protocol/sei-chain/x/evm/keeper"
+	"github.com/sei-protocol/sei-chain/x/evm/types"
+	"github.com/sei-protocol/sei-chain/x/evm/types/ethtx"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEVMSigVerifyDecorator(t *testing.T) {
```

Comment:
let's also add tests for non-existent sei<>evm addrs (if not already covered elsewhere)

Reply:
added